pyface/data_view/i_data_view_widget.py

```diff
@@ -37,7 +37,7 @@ class IDataViewWidget(IWidget):
     #: handle drop actions which either affect the whole data view, or where
     #: the data handler can work out how to change the underlying data without
     #: additional input.
-    drop_handlers = List(Instance(IDropHandler))
+    drop_handlers = List(Instance(IDropHandler, allow_none=False))
```

Comment:
IIUC, this ensures that it's an empty list of a list of handlers - not a list of `None`s